src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -58,11 +58,6 @@ private static void VerifyParams(IParameterSymbol parameter, bool isParamArray =
             VerifyParams(parameter.GetSymbol<ParameterSymbol>(), isParamArray: isParamArray, isParamCollection: isParamCollection);
         }
 
-        private static string ExpectedOutput(string output)
```

Comment:
>ExpectedOutput [](http://example.com/codeflow?start=30&length=14)

I do not think moving this method to a shared location with this name an implementation is appropriate. The name is too general/unspecific and the logic is not applicable in every test.